server-x/tools/docker-entrypoint.sh

```diff
@@ -38,48 +38,6 @@ file_env 'KEYCLOAK_ADMIN_PASSWORD'
 # Start Keycloak #
 ##################
 
-CONFIG_ARGS=""
-RUN_CONFIG_START=false
-RUN_CONFIG=false
-SERVER_OPTS=""
-
-while [ "$#" -gt 0 ]
-do
-    case "$1" in
-      --auto-config)
-          RUN_CONFIG_START=true
-          ;;
-      config)
-          RUN_CONFIG=true
-          ;;
-      *)
-          if [[ $1 = --* || ! $1 =~ ^-.* ]]; then
-            CONFIG_ARGS="$CONFIG_ARGS $1"
-          else
-            SERVER_OPTS="$SERVER_OPTS $1"
-          fi
-          ;;
-    esac
-    shift
-done
-
-if [[ "$RUN_CONFIG_START" == true ]]; then
-  if [[ "$RUN_CONFIG" == true ]]; then
-    echo "ERROR: You can not run 'config' when passing the '--auto-config' start option.Please, choose one or another."
-    exit 2
-  fi
-
-  CONFIG_MARKER_FILE="/opt/jboss/keycloak/config_marker"
-
-  if [[ -n $CONFIG_ARGS && ! -f "$CONFIG_MARKER_FILE" ]]; then
-    exec /opt/jboss/keycloak/bin/kc.sh config $CONFIG_ARGS &
-    wait $!
-    touch $CONFIG_MARKER_FILE
-  fi
-
-  exec /opt/jboss/keycloak/bin/kc.sh $SERVER_OPTS
-else
-  exec /opt/jboss/keycloak/bin/kc.sh $SERVER_OPTS $CONFIG_ARGS
-fi
+exec /opt/keycloak/bin/kc.sh $@
```

Comment:
Do we even need a "docker-entrypoint.sh" file?

Reply:
cc. @pedroigor 
we still have this logic encoded there:
https://github.com/keycloak/keycloak-containers/blob/f8674774f6852110c4e64a30815da770374fa7c9/server-x/tools/docker-entrypoint.sh#L4-L35

Reply:
@stianst It is only there because it seems to be related to a docker feature. To be honest, I don't know how it works but it is there in the wildfly image.

If you think it does not make sense anymore, we can definitely remove the entrypoint script.

Reply:
We have the "file" vault right which allows mounting secrets into files? I'd rather align with that then have these separate docker image only env variables (that won't be documented automatically as we're listing properties from Keycloak source code).

Reply:
Sounds good to me.